src/ui/utils/environment.ts

```diff
@@ -98,6 +98,18 @@ export function hasLoadingParam() {
   return getURL().searchParams.get("loading") != null;
 }
 
+export function getFocusRegion() {
+  const url = getURL();
+  const focusRegionParam = url.searchParams.get("focusRegion");
+  try {
+    return focusRegionParam
+      ? (decodeBase64FromURL(focusRegionParam) as TimeStampedPointRange)
+      : undefined;
+  } catch (e) {
+    return undefined;
```

Comment:
This _used_ to be `null` and now it's `undefined`. I don't know if that matters or not but it looks like a change in behavior?